Framework/PythonInterface/plugins/algorithms/CreateCacheFilename.py

```diff
@@ -61,7 +61,9 @@ def PyInit(self):
 
         self.declareProperty("OutputFilename", "", "Full path of output file name", Direction.Output)
 
-        self.declareProperty("OutputSignature", "", "Calculated sha1 hash", Direction.Output)
+        self.declareProperty("OutputSignature", "",
+                             "Comma-separated sorted list of key=values pairs used to create the sha1 hash",
+                             Direction.Output)
```

Comment:
Actually the desired output for `OutputSignature` should be the SHA1 value instead of the "key=values" pairs.
So the modification should be in the `PyExec()` where the SHA1 string instead of the "key=value"-pair string is set to `OutputSignature".

Reply:
fixed